src/tribol/mesh/MfemData.cpp

```diff
@@ -901,9 +1140,15 @@ MfemJacobianData::UpdateData::UpdateData( const MfemMeshData& parent_data, const
   }
   // create a matrix transfer operator for moving data from redecomp to the
   // submesh
-  submesh_redecomp_xfer_ = std::make_unique<redecomp::MatrixTransfer>( *dual_submesh_fes, *primal_submesh_fes,
-                                                                       *submesh_data.GetRedecompGap().FESpace(),
-                                                                       *parent_data.GetRedecompResponse().FESpace() );
+  submesh_redecomp_xfer_00_ = std::make_unique<redecomp::MatrixTransfer>(
```

Comment:
Looks like this routine now has three transfer operators whereas it used to only have one. Can you comment?

Reply:
The "row" and "column" space of the transfer operator changes depending on which block we are transferring.  Before, we just transferred off diagonal blocks, and further, the matrix was symmetric, so 1 operator sufficed.